graphql/data/schema.js

```diff
@@ -746,6 +749,18 @@ const userType = new GraphQLObjectType({
       description: 'feature values for this specific user',
       resolve: (user, args, context) => getUserFeatures(context, user),
     },
```

Comment:
Related: can you change the return type for setUserSearchEngineMutation? That way, the refetched data will update the client behavior appropriately.

Reply:
(Sorry for not thinking to call that out last time.)